lms/templates/fields/field_textarea.underscore

```diff
@@ -27,8 +27,12 @@
             <% } %>
             ><%- value %></span>
         <% } else { %>
-            ><a href="#"><p class="sr"><%- screenReaderTitle %></p><span class="u-field-value-readonly" aria-hidden="false" aria-describedby="u-field-placeholder-value-<%- id %>"><%- value %></span><span class="sr"><%- gettext('Click to edit') %></span></a>
+            ><span class="clickable" role="button" tabindex="0">
```

Comment:
@clrux Did you wish to leave these changes for that other JIRA ticket you created?